server/server.js

```diff
@@ -3,6 +3,7 @@ require('dotenv').config();
 const path = require('path');
 const express = require('express');
 const morgan = require('morgan');
```

Comment:
I noticed this code `app.use(morgan('tiny'));` you deleted. Can you explain what's meaning of it? 
If we don't need morgan right now, do we need to commit out it first?